lib/cron_kubernetes/context/kubectl.rb

```diff
@@ -34,7 +34,7 @@ def auth_options(config)
       end
 
       def google_application_default_credentials
-        return unless defined?(Google) && defined?(Google::Auth)
+        return unless defined?(Google::Auth)
```

Comment:
It seems this could still fail if you don't have `Google` defined in your environment